src/pointwise_loglikelihoods.jl

```diff
@@ -0,0 +1,40 @@
+function _pd_diag_inv(A::PDMats.AbstractPDMat)
+    T = typeof(float(oneunit(eltype(A))))
+    I = Diagonal(ones(T, axes(A, 1)))
+    return PDMats.invquad(A, I)
+end
+
+function pointwise_loglikelihood!(
+    logl::AbstractVector{<:Real}, y::AbstractVector{<:Real}, dist::Distributions.MvNormal
```

Comment:
For clarity, let's rename `logl` to `log_like` for consistency with our `loo` implementation:
```suggestion
    log_like::AbstractVector{<:Real},
    y::AbstractVector{<:Real},
    dist::Distributions.MvNormal,
```